backend/src/test/java/com/ody/meeting/repository/MeetingRepositoryTest.java

```diff
@@ -132,4 +133,34 @@ void findAllByUpdatedTodayAndOverdue() {
         assertThat(meetingRepository.findAllByUpdatedTodayAndOverdue()).extracting(Meeting::getId)
                 .containsExactly(oneDayBeforeMeeting.getId(), twoDayBeforeMeeting.getId());
     }
+
+    @DisplayName("약속 기한이 지나지 않은 모임방을 조회한다.")
+    @Test
+    void findByIdAndOverdueFalse() {
+        LocalDateTime dateTime = TimeUtil.nowWithTrim();
+        Meeting wotecoMeeting = new Meeting(
+                null,
+                "우테코 등교",
+                dateTime.toLocalDate(),
+                dateTime.toLocalTime(),
+                TARGET_LOCATION,
+                "초대코드",
+                true
+        );
+        Meeting savedWotecoMeeting = meetingRepository.save(wotecoMeeting);
+
+        Meeting odyMeeting = new Meeting(
+                null,
+                "오디 회식",
+                dateTime.toLocalDate(),
+                dateTime.toLocalTime(),
+                TARGET_LOCATION,
+                "초대코드",
+                false
+        );
+        Meeting savedOdyMeeting = meetingRepository.save(odyMeeting);
+
+        assertThat(meetingRepository.findByIdAndOverdueFalse(savedWotecoMeeting.getId())).isEmpty();
+        assertThat(meetingRepository.findByIdAndOverdueFalse(savedOdyMeeting.getId())).isPresent();
+    }
```

Comment:
[제안] 어떤 약속인지 드러나면(overdueMeeting, notOverdueMeeting) 코드가 더 눈에 잘 들어올 것 같아요!!

```suggestion
    @DisplayName("약속 기한이 지난/지나지 않은 모임방을 조회한다.")
    @Test
    void findByIdAndOverdueFalse() {
        boolean isOverdue = true;
        Meeting overdueMeeting = meetingRepository.save(generateMeeting(isOverdue));
        Meeting notOverdueMeeting = meetingRepository.save(generateMeeting(!isOverdue));

        assertThat(meetingRepository.findByIdAndOverdueFalse(savedOverdueMeeting.getId())).isEmpty();
        assertThat(meetingRepository.findByIdAndOverdueFalse(savedNotOverdueMeeting.getId())).isPresent();
    }

    private Meeting generateMeeting(boolean overdue) {
        LocalDateTime dateTime = TimeUtil.nowWithTrim();

        return new Meeting(
                null,
                "우테코 등교",
                dateTime.toLocalDate(),
                dateTime.toLocalTime(),
                TARGET_LOCATION,
                "초대코드",
                overdue
        );
    }
```

Reply:
mzeong 합니다 👍 
어떤 약속인지 드러나는게 더 명확할 것 같아요
메서드로 빼는 것은 추후에 전체적으로 추상 클래스에서 공통화할 수 있을 것 같아 변수명만 바꿨습니다 !